webapp/sources/rudder/rudder-web/src/main/elm/rules/sources/ViewTabContent.elm

```diff
@@ -294,20 +295,21 @@ directivesTab model details =
       Just (_,_,sortFun) -> (\i1 i2 -> sortFun (fun.data model i1) (fun.data model i2))
       Nothing -> (\_ _ -> EQ)
     filter       = model.ui.directiveFilters.tableFilters.filter
-    childs       = Maybe.withDefault [] (Maybe.map .directives details.compliance)
-    childrenSort = childs
+    childs       = Maybe.withDefault [] (Maybe.map .directives (log "details.compliance" details.compliance))
+    childrenSort = (log "childs" childs)
       |> List.filter (\d -> (filterSearch model.ui.directiveFilters.tableFilters.filter (searchFieldDirectiveCompliance d)))
       |> List.sortWith sort
     (directivesChildren, order, newOrder) = case sortOrder of
-       Asc -> (childrenSort, "asc", Desc)
+       Asc -> ((log "childrenSort" childrenSort), "asc", Desc)
        Desc -> (List.reverse childrenSort, "desc", Asc)
 
     ruleDirectives = directives
       |> List.filter (\d -> List.member d.id ruleDirectivesId)
 
     disabledRuleDirectives = ruleDirectives
       |> List.filter (\d -> not d.enabled)
-
+    skippedDirectives = ruleDirectives
+      |> List.filter (\d -> (not (List.member d.id.value (List.map (\f -> f.directiveId.value) directivesChildren))) && d.enabled)
```

Comment:
Here I retrieve the list of directives that are enabled and that are absent from the compliance list of directives. In this bug, the problem is that the directive skipped is not part of the response of the API `getRulesComplianceId`.
But we have the skipped directive when we get the directive tree library.

So we can guess that if the directive is skipped:
- Enable
- present in directive tree
- linked to the `ruleId`
- absent from the directives compliance list of `ruleId`